packages/next/src/withPayload.js

```diff
@@ -46,7 +46,7 @@ export const withPayload = (nextConfig = {}) => {
       return [
         ...(headersFromConfig || []),
         {
-          source: '/:path*',
+          source: '/admin/:path*',
```

Comment:
Consider that you can have a different admin route path https://payloadcms.com/docs/beta/admin/overview#root-level-routes

Reply:
Good catch @r1tsuu. I just updated to use the `adminRoute` from the `payloadConfig` if it's been defined.

Reply:
since it's an untyped JS file, users won't really know that it can accept `adminRoute`
`{ adminRoute = '/admin' } = {}` should work, either way is to put a JSDoc.

Reply:
I just made that update @r1tsuu